man/roxygen/meta.R

```diff
@@ -0,0 +1,14 @@
+list(
+  rd_family_title = list(
+    der_adxx = "ADXX Functions that returns variable appended to dataset: ",
+    der_prm_adxx = "ADXX Functions for adding Parameters: ",
+    com_adxx = "ADXX Functions that returns a vector: ",
+    utils_ds_chk = "Utilities for Dataset Checking: ",
+    utils_fil = "Utilities for Filtering Observations: ",
+    utils_fmt = "Utilities for Formatting Observations: ",
+    utils_help = "Utilities used within Derivation functions: ",
+    utils_examples = "Utilities used for examples and template scripts: ",
+    source_specifications = "Source Specifications: ",
+    high_order_function = "Higher Order Functions: "
+  )
+)
```

Comment:
This is great!!  Thanks for adding this

Reply:
Could we unify the case?

Reply:
No worries! @bundfussr for your point "Could we unify the case?" I think that could come as a separate nice-to-have issue later as it would need fixing in admiral core, onco etc too. I just matched admiral for now